contracts/eos/BancorConverter/src/convert.cpp

```diff
@@ -74,22 +72,22 @@ std::tuple<asset, double> BancorConverter::calculate_return(extended_asset from_
     }
     else {
         if (!incoming_smart_token) { // Reserve --> Smart
```

Comment:
can basically completely remove support for swaps between reserve <> smart token.
smart tokens are acquired by adding liquidity.
also, "smart token" isn't used anymore - it's just token (no need to migrate for this :) but would be helpful to rename wherever possible that doesn't require migration)

Reply:
Ok, good to know, perhaps we can create a new PR for everything that's `smart token` removal

Does this mean that the convert method only handles `quick_conversion`?

Reply:
It's actually out of the scope of this migration so we'll handle that separately later.